operator/docs/operations.md

```diff
@@ -1,3 +1,45 @@
+
+More information on Pipeline Service is provided [here](../../README.md).
+ 
+Development Mode: To facilitate the usage of Pipeline Service, developers and SREs can easily try out the software by running it in Development mode on any OpenShift or Kubernetes cluster. Instructions and details on how to use the Development mode are available [here](../../developer).
+
+### Security
+
+Pipeline Service takes security seriously and strives to provide a secure platform. Pipeline Service enforces security measures through a combination of best practices and tooling, which are used to identify and address potential security issues. The team follows established security best practices, including regular updates to dependencies, strong authentication and authorization mechanisms, and secure communication channels. In addition, various tools are used to scan the codebase for potential security vulnerabilities, including SAST checks and vulnerability scanning.
+
+#### SAST Checking & Linters
+Pipeline Service uses Static Application Security Testing (SAST) to scan the codebase for potential security vulnerabilities. This is done using various tools such as SonarQube, Bandit, and other open-source tools. The SAST scans are run automatically as part of the Continuous Integration (CI) process, and results are reported back to the development team. 
+
+The below tools are currently used in Pipeline Service: 
+- [checkov](https://github.com/bridgecrewio/checkov)
+- [hadolint](https://github.com/hadolint/hadolint)
+- [shellcheck](https://github.com/koalaman/shellcheck)
+- [yamllint](https://github.com/adrienverge/yamllint)
+
+
+#### Vulnerability Scanning
+
+Pipeline Service also employs vulnerability scanning to detect and address potential security issues in its dependencies. This is done using Clair. The vulnerability scans are run regularly, and the results are analyzed by the team to identify any potential risks or issues.
+
+### Tests
+
+Pipeline Service employs a rigorous testing process to ensure the quality and reliability of the service. We run tests on every PR as part of the CI process to catch any issues early in the development cycle. As part of each PR, we run SAST checks to identify potential security vulnerabilities and functional tests to ensure that Pipeline Service remains intact.
+
+We use Tekton Pipelines to run our tests. All our tasks, pipelines, and pipelineruns live in the [.tekton folder](../../.tekton) in our repository. This approach allows us to define the test pipelines in a declarative way, making it easy to maintain and modify. In the future, we plan to dogfood Pipeline Service by using it to test itself. This approach also provides an opportunity to identify any potential issues with the service itself and address them before they affect our users.
+
+### Upgrade Strategy for Operators
```

Comment:
Probably better to use a more general term like "Deployment" since it is not only about upgrades here.

Reply:
Sorry, not sure I get your point here. This specific section of the doc is to talk about our upgrade strategy.